robot.py

```diff
@@ -15,24 +19,65 @@ def __init__(self, x: float, y: float, radius: float, anglex: float):
         self.radius = radius
         self.anglex = anglex
 
+class EllipseDefinition:
+    x: float
+    y: float
+    d1: float
+    d2:float
+    theta:float
+    anglex: float
+
+    def __init__(self, x: float, y: float, d1: float, d2:float, theta:float, anglex: float):
+        self.x = x
+        self.y = y
+        self.d1 = d1
+        self.d2 = d2
+        self.theta = theta
+        self.anglex = anglex
+
 
 class RobotConnection:
     def __init__(self, robot_ip: str, fake: bool=False):
         self.robot_ip = robot_ip
         self.fake = fake
 
+        
+
```

Comment:
nit: extraneous spacing. Class members should only have a single line between each other